fabric-screen-handler-api-v1/src/main/java/net/fabricmc/fabric/impl/screenhandler/client/NetworkingClient.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.screenhandler.client;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.screen.ingame.HandledScreens;
+import net.minecraft.client.gui.screen.ingame.ScreenHandlerProvider;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.network.PacketByteBuf;
+import net.minecraft.screen.ScreenHandlerType;
+import net.minecraft.text.Text;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.network.ClientSidePacketRegistry;
+import net.fabricmc.fabric.impl.screenhandler.ExtendedScreenHandlerType;
+import net.fabricmc.fabric.impl.screenhandler.Packets;
+
+@Environment(EnvType.CLIENT)
+public final class NetworkingClient implements ClientModInitializer {
+	private static final Logger LOGGER = LogManager.getLogger();
+
+	@Override
+	public void onInitializeClient() {
+		ClientSidePacketRegistry.INSTANCE.register(Packets.OPEN_ID, (ctx, buf) -> openScreen(buf));
+	}
+
+	@SuppressWarnings({"rawtypes", "unchecked"})
+	private void openScreen(PacketByteBuf buf) {
+		int typeId = buf.readVarInt();
+		int syncId = buf.readVarInt();
+		Text title = buf.readText();
+		ScreenHandlerType<?> type = Registry.SCREEN_HANDLER.get(typeId);
+
+		if (type == null) {
+			LOGGER.warn("[Fabric] Unknown screen handler ID: {}", typeId);
```

Comment:
Unsure about putting the `[Fabric]` in the logs directly. I want to get someone else to look at this as a standards question as well.

Reply:
Good spotting. The logger name should suffice; this `[Fabric]` should be removed.